src/Symfony/Component/Webhook/Exception/RejectWebhookException.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Webhook\Exception;
+
+use Symfony\Component\HttpKernel\Exception\HttpException;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @experimental in 6.3
+ */
+class RejectWebhookException extends HttpException
+{
+    public function __construct(int $statusCode = 406, string $message = '', \Throwable $previous = null, array $headers = [], int $code = 0)
```

Comment:
We never set the `$statusCode` to anything but `406`. But I understand that allowing to change the status code makes sense nevertheless. But maybe we should change the parameter order.

```suggestion
    public function __construct(string $message = '', \Throwable $previous = null, int $statusCode = 406, array $headers = [], int $code = 0)
```

Reply:
I would keep the same order as the one for `HttpException`.